mantidimaging/core/data/imagestack.py

```diff
@@ -349,6 +349,16 @@ def shutter_count_file(self, value: ShutterCount | None) -> None:
             del self.metadata[const.SHUTTER_COUNT_FILE]
         self._shutter_count_file = value
 
+    def find_image_from_angle(self, target_angle: float, tol: float | None = None) -> int:
```

Comment:
There needs to be some unit conversion somewhere, maybe in this function. The ImageStack holds the angles in radians, but we want the user to be choosing in degrees.